### ISSUE TITLE
Auto save state not working on exit

### ISSUE THREAD

**Issue by fr500** [libretro/RetroArch#3049]
As the title reads, auto save state not working on RetroArch exit, it is working on content close though.

**Comment by TheCherry** [comment]
had that problem too, did fix it by adding the save path to the commandline. -S and -s

**Comment by fr500** [comment]
Actually those options have their own share of problems. And no that is not
it

**Comment by twinaphex** [comment]
Should work now I think - please close if you can confirm it's fixed, it worked for me so far at least.

https://github.com/libretro/RetroArch/commit/dc4275b67344f3663ee8b52e6f35ff5e101cca80

**Comment by pantra64** [comment]
I'm experiencing this in RetroArch 1.3.6 with the Genesis Plus GX core. But someone else seemed to have troubles as well over here: [http://libretro.com/forums/showthread.php?t=6379](url)

**Comment by bparker06** [comment]
You might be experiencing issue #3335 which was not fixed until after 1.3.6. Can you try the latest nightly build and see if it fixes your issue?

**Comment by pantra64** [comment]
@bparker06 Thank you very much, it's working with the current nightly!

**Comment by BetaLeaf** [comment]
I just lost over an hours worth of progress in my game when I accidentally hit the ESC key. My AutoSave is on so the game should have saved, however, when I reloaded my game, it didn't AutoLoad is also on. I disabled the Exit on ESC setting so I won't accidentally close my game until im sure theres a savestate. 

TL;DR: It's Broken again on Windows 10.

**Comment by fr500** [comment]
> RetroArch 1.5.0.
this is old

**Comment by BetaLeaf** [comment]
Ok, now I can't play any of my games. It says "Failed to open libretro core" and my debug log says 
```
RetroArch [ERROR] :: Failed to open libretro core: "D:\OneDrive\PortableApps\RetroArchPortable\App\RetroArch\cores\gambatte_libretro.dll"
RetroArch [ERROR] :: Error(s): %1 is not a valid Win32 application.
```
I feel I should open a new issue at this point. If you wanted to help, then check the latest issues in like 2 minutes.
